File: Week1/Week1CS/Week1/Credits.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Week1{
    public partial class Credits : Form{
        public Credits(){
            InitializeComponent();
            label1.Text = "This application was written by" +
                "\nAlessandro Albini (AxeR)" +
                "\n(ID 2032013)" +
                "\n Github: https://github.com/AxeR44/statistics_2223" +
                "\n Website: https://axer44.github.io/statistics_2223/";
            label1.TextAlign = ContentAlignment.MiddleCenter;
            string pwd = Directory.GetCurrentDirectory();
            Bitmap bmp = new Bitmap(".\\assets\\sapienza.png");
            pictureBox1.SizeMode = PictureBoxSizeMode.StretchImage;
            pictureBox1.Image = bmp;
        }
    }
}

File: Week2/VB.NET/RandomTimer/RandomTimer/Credits.vb
Imports System.IO

Public Class Credits
    Private Sub Credits_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        Me.label1.Text = "This application was written by" + System.Environment.NewLine +
                "Alessandro Albini (AxeR)" + System.Environment.NewLine +
                "(ID 2032013)" + System.Environment.NewLine +
                "Github: https://github.com/AxeR44/statistics_2223" + System.Environment.NewLine +
                "Website: https://axer44.github.io/statistics_2223/"
        Me.label1.TextAlign = ContentAlignment.MiddleCenter
        Dim pwd As String
        pwd = Directory.GetCurrentDirectory()
        Dim bmp = New Bitmap(".\\assets\\sapienza.png")
        Me.pictureBox1.SizeMode = PictureBoxSizeMode.StretchImage
        pictureBox1.Image = bmp
    End Sub
End Class